src/mailchimp/mailchimp_manager.py

```diff
@@ -48,8 +54,14 @@ def set_authorisation(self, keys: Dict[str, str]) -> bool:
         Raises:
             TODO: Document all possible errors
         """
-        raise NotImplementedError
-    
+	try:
+		keys["server"] = self.server
+		keys["api_key"] = self.api_key
+	except ValueError:
+		print("parameters given are invalid")
+	except TypeError:
+		print("missing parameters")
```

Comment:
The Mailchimp manager is not responsible for user IO. If an error occurs, then it should be raised, not printed. Also, it's not possible for a `ValueError` nor a `TypeError` to occur here as it's not being raised. Lines 58-59 may raise an `AttributeError` if `keys` does not support subscripting or a `TypeError` if `"server"` or `"api_key"` are not found.